src/util/errors.js

```diff
@@ -1,13 +1,13 @@
 import { ERROR_TYPE } from "../constants/ErrorTypes";
 
-export function renderErrorMessage(errorType) {
+export function renderErrorMessage(errorType, eventsUrl) {
```

Comment:
This is not a huge deal, but it feels weird for a function that renders an error message to get an `eventsUrl`. Maybe it could instead accept an appId and deviceId and derive the URL?

Reply:
Yeah, good point. I refactored so only the device UID needs to be passed in now, which also eliminates the need to share `eventsUrl` around the app. 👍